### PR TITLE
Doc: add known errors section for the plugin timeout error

### PR DESCRIPTION
**Pull request by RobinSCU**
Hi,

I added the Known Errors Section as discussed here: https://github.com/fastify/fastify-nextjs/issues/464

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,6 +175,20 @@ This project is kindly sponsored by:
 - [NearForm](https://nearform.com)
 - [LetzDoIt](https://www.letzdoitapp.com/)
 
+## Known Errors
+### Plugin Timeout
```
Comment (darkgl0w) [inline README.md:179]:
```suggestion
## Plugin Timeout and Next.js development mode
```

Comment (darkgl0w) [reply README.md:179]: Nit.

**Review by darkgl0w — COMMENTED** [review]
Can you put this section before the `Development` one please ?

**Comment by darkgl0w** [comment]
@RobinSCU > can you move it before the Development section ?

It is better if we have the sections in this order:
- Install
- Usage
- Custom properties on the request object
- Plugin Timeout and Next.js development mode
- Development
- Acknowledgements
- License

**Review by darkgl0w — APPROVED** [review]

**Review by Eomm — APPROVED** [review]

**Pull request merged**